pcsx2/gui/AppInit.cpp

```diff
@@ -167,16 +167,25 @@ void Pcsx2App::AllocateCoreStuffs()
 			{
 				scrollableTextArea->AppendText( L"* microVU0\n\t" + ex->FormatDisplayMessage() + L"\n\n" );
 				recOps.UseMicroVU0	= false;
+#ifndef DISABLE_SVU
 				recOps.EnableVU0	= recOps.EnableVU0 && m_CpuProviders->IsRecAvailable_SuperVU0();
+#else
+				recOps.EnableVU1	= false;
```

Comment:
recOps.EnableVU0 => mean use a dynarec for vu0 (otherwise interpreter)
In case of dynarec, EmuConfig.Cpu.Recompiler.UseMicroVU0 select MicroVu or SuperVu